langwatch/src/pages/api/triggers/index.ts

```diff
@@ -130,7 +147,7 @@ const getTracesForAlert = async (trigger: Trigger, projects: Project[]) => {
 
     if (action === TriggerAction.SEND_EMAIL) {
       triggerInfo = {
-        triggerEmails: (actionParams as ActionParams)?.members ?? [],
+        triggerEmails: (actionParams as unknown as ActionParams)?.members ?? [],
```

Comment:
hmm why was it necessary to add unknown here now?

Reply:
so it brought up a typscripts error, and it suggested this, to went with it